release-lockfile.spec.md

```diff
@@ -221,7 +249,7 @@ Sources are declared in a key/value mapping.
 * Values **must** conform to *one of* the following formats.
     * Source string.
         * When the value is a source string the key should be interpreted as a file path.
-    * IPFS URI
+    * Content Addressable URI
```

Comment:
💯

Reply:
Did we want to reference rfc3986 at all? It's a pretty definitive document on URI's but may be much, just throwing it out there.

Reply:
I'll get a link to that RFC added.